src/Grids/AbstractResourceGrid.php

```diff
@@ -155,13 +150,14 @@ public function exportCsv($data)
             $csvData[] = implode(',', $itemsArray);
         }
 
+        $csvText = '';
         foreach ($csvData as $row) {
-            fputcsv($handle, [$row]);
+            $csvText .= $row . "\n";
```

Comment:
There is a PHP constant that represents the end of the line

`PHP_EOL`

```suggestion
            $csvText .= $row . PHP_EOL;
```